packages/@webex/plugin-cc/src/services/WebCallingService.ts

```diff
@@ -1,27 +1,32 @@
+import EventEmitter from 'events';
 import {
   createClient,
   ICall,
   ICallingClient,
   ILine,
   LINE_EVENTS,
   CallingClientConfig,
+  LocalMicrophoneStream,
 } from '@webex/calling';
-import {WebexSDK} from '../types';
+import {LoginOption, WebexSDK} from '../types';
 import {TIMEOUT_DURATION, WEB_CALLING_SERVICE_FILE} from '../constants';
 import LoggerProxy from '../logger-proxy';
 
-export default class WebCallingService {
+export default class WebCallingService extends EventEmitter {
   private callingClient: ICallingClient;
   private callingClientConfig: CallingClientConfig;
   private line: ILine;
   private call: ICall;
   private webex: WebexSDK;
+  public loginOption: LoginOption;
   constructor(webex: WebexSDK, callingClientConfig: CallingClientConfig) {
+    super();
     this.webex = webex;
     this.callingClientConfig = callingClientConfig;
   }
 
-  public async registerWebCallingLine(): Promise<void> {
+  public async registerWebCallingLine(loginOption: LoginOption): Promise<void> {
+    this.loginOption = loginOption;
```

Comment:
we don't seem to be using this anywhere. Is this needed?

Reply:
We are using this in app.js

Reply:
This is used in TaskManager